deepchem/feat/graph_features.py

```diff
@@ -953,145 +956,144 @@ def _featurize(self, mol):
-    strip_hydrogens: bool
-      Whether to remove hydrogens while computing neighbor features
-    learning_rate: float
-      Learning rate for training the model
-    epochs: int
-      Number of epochs to train the model for
-    """
-
-    self.atomic_conv_model = dc.models.atomic_conv.AtomicConvModel(
-        frag1_num_atoms=frag1_num_atoms,
-        frag2_num_atoms=frag2_num_atoms,
-        complex_num_atoms=complex_num_atoms,
-        max_num_neighbors=max_num_neighbors,
-        batch_size=batch_size,
-        atom_types=atom_types,
-        radial=radial,
-        layer_sizes=layer_sizes,
-        learning_rate=learning_rate)
-
-    super(AtomicConvFeaturizer, self).__init__(
-        frag1_num_atoms=frag1_num_atoms,
-        frag2_num_atoms=frag2_num_atoms,
-        complex_num_atoms=complex_num_atoms,
-        max_num_neighbors=max_num_neighbors,
-        neighbor_cutoff=neighbor_cutoff,
-        strip_hydrogens=strip_hydrogens)
-
-    self.epochs = epochs
-    self.labels = labels
-
-  def featurize(self, mol_files, protein_files):
-    features = []
-    failures = []
-    for i, (mol_file, protein_pdb) in enumerate(zip(mol_files, protein_files)):
-      logging.info("Featurizing %d / %d" % (i, len(mol_files)))
-      new_features = self._featurize(mol_file, protein_pdb)
-      # Handle loading failures which return None
-      if new_features is not None:
-        features.append(new_features)
-      else:
-        failures.append(ind)
-
-    features = np.asarray(features)
-    labels = np.delete(self.labels, failures)
-    dataset = DiskDataset.from_numpy(features, labels)
-
-    # Fit atomic conv model
-    self.atomic_conv_model.fit(dataset, nb_epoch=self.epochs)
-
-    # Add the Atomic Convolution layers to fetches
-    layers_to_fetch = [
-        self.atomic_conv_model._frag1_conv, self.atomic_conv_model._frag2_conv,
-        self.atomic_conv_model._complex_conv
-    ]
-
-    # Extract the atomic convolution features
-    atomic_conv_features = list()
-    batch_generator = self.atomic_conv_model.default_generator(
-        dataset=dataset, epochs=1)
-
-    for X, y, w in batch_generator:
-      frag1_conv, frag2_conv, complex_conv = self.atomic_conv_model.predict_on_generator(
-          [(X, y, w)], outputs=layers_to_fetch)
-      concatenated = np.concatenate(
-          [frag1_conv, frag2_conv, complex_conv], axis=1)
-      atomic_conv_features.append(concatenated)
-
-    batch_size = self.atomic_conv_model.batch_size
-
-    if len(features) % batch_size != 0:
-      num_batches = (len(features) // batch_size) + 1
-      num_to_skip = num_batches * batch_size - len(features)
-    else:
-      num_to_skip = 0
-
-    atomic_conv_features = np.asarray(atomic_conv_features)
-    atomic_conv_features = atomic_conv_features[-num_to_skip:]
-    atomic_conv_features = np.squeeze(atomic_conv_features)
-
-    return atomic_conv_features, failures
+#class AtomicConvFeaturizer(ComplexNeighborListFragmentAtomicCoordinates):
```

Comment:
This is commented out since we renamed `AtomicConvFeaturizer` in DeepChem 2.5 to the standard featurizer for atomic convolutions. This is an older featurizer that uses atomic conv models to make embeddings that doesn't currently work. I think we should go ahead and comment this out for now since it's broken and not currently exposed at the DeepChem top level